stack/core/src/main/java/org/apache/usergrid/corepersistence/CpSetup.java

```diff
@@ -202,17 +201,17 @@ public void setupStaticKeyspace() throws Exception {
 
         if ( USE_VIRTUAL_KEYSPACES ) {
```

Comment:
This if isn't needed anymore, we don't always use virtual.  Every row key is app prefixed, and we put all systems into the same keyspace. Eventually, we'll move to an org->keyspace lookup system.